src/routes/[lang]/contact/christian/+page.svelte

```diff
@@ -0,0 +1,74 @@
+<script lang="ts">
+  import { browser } from '$app/environment'
+  import googleWallet from '$lib/assets/google-wallet.svg'
+  import iosWallet from '$lib/assets/ios-wallet.svg'
+
+  export let data
+  const { passGoogle, pkpass } = data
+
+  let isIOS: boolean
+  let isAndroid: boolean
+  let isMac: boolean
+
+  if (browser) {
+    const { userAgent } = navigator
+    isIOS = /(iPhone|iPad|iPod)/.test(userAgent)
+    isAndroid = /(Android)/.test(userAgent)
+    isMac = /(Macintosh)/.test(userAgent)
+
+    if (isIOS || isMac) {
+      window.location.href = pkpass
+    } else if (isAndroid) {
+      window.location.href = passGoogle
+    }
+  }
+</script>
+
+<div class="add-wallet">
+  <a href={pkpass}><img class="ios" src={iosWallet} alt="button" /></a>
+  <a href={passGoogle}
+    ><img class="android" src={googleWallet} alt="button" /></a
+  >
+</div>
+
+<style>
+  .add-wallet {
+    gap: 50px;
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+  .android {
+    width: 320px;
+    height: 89px;
```

Comment:
magic numbers - why not just height: auto ? 

and then they can even share the class

Reply:
well no, it is calculated with the reference size that the download svg gives you

Reply:
auto is better